import_ldraw.py

```diff
@@ -318,18 +326,19 @@ def parse(self, filename):
                                 x, y, z, a, b, c,
                                 d, e, f, g, h, i
                             ) = map(float, tmpdate[2:14])
-                            mat_new = self.mat * mathutils.Matrix((
-                                    (a, b, c, x),
-                                    (d, e, f, y),
-                                    (g, h, i, z),
-                                    (0, 0, 0, 1)
-                                ))
-
+                            # - Reset orientation of top-level Part, track original orientation
+                            if self.part_count == 1 and isPart:
+                                mat_new = self.mat * mathutils.Matrix(((1, 0, 0, 0), (0, 1, 0, 0), (0, 0, 1, 0), (0, 0, 0, 1)))
```

Comment:
Matrixes are expanded out because they break the 80 character line limit. If you need help expanding these, let me know.